src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/ReportsController.kt

```diff
@@ -222,11 +234,17 @@ class ReportsController(
 
     val properties = Cas1PlacementMatchingOutcomesReportProperties(year, month)
 
-    return generateXlsxStreamingResponse { outputStream ->
+    return generateStreamingResponse(
+      contentType = ContentType.XLSX,
+      fileName = createCas1ReportName("placement-matching-outcomes", year, month, ContentType.XLSX),
+    ) { outputStream ->
       cas1ReportService.createPlacementMatchingOutcomesReport(properties, outputStream)
     }
   }
 
+  private fun createCas1ReportName(name: String, year: Int, month: Int, contentType: ContentType) =
+    "$name-$year-${month.toString().padStart(2, '0')}.${contentType.extension}"
```

Comment:
just an idea but I think users can find it helpful if downloads are timestamped. They often run the same report multiple times and can get in a muddle identifying the "right" version. e.g.

`placement-matching-outcomes-2024-june-20240604_0942.xlsx`

Reply:
Good idea, i'll do that before merging